### PR TITLE
Abbas-01-html

### PR DESCRIPTION
**Pull request by AbbasShaikh0304**
html Basic Project

### CODE REVIEW

File: tasks-by-users/AbbasShaikh0304/01-Project/Style.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.Heading {
+    color: orangered;
+    text-align: center;
+    font-family: Arial, Helvetica, sans-serif;
+    background-image: none;
+}
+.Genre {
+    display: block;
+}
```
Comment (AlishaPrasad) [inline tasks-by-users/AbbasShaikh0304/01-Project/Style.css:9]: Where is this css used in the code? I dont see any Heading or Genre classes added in html files.

File: tasks-by-users/AbbasShaikh0304/01-Project/index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        genre at most affordable prize. You can also download the digital copy of
+        your Books.
+    </p>
+
+    <h2>Genre</h2>
+    <hr />
+    <ul>
+
+        <li>
+            <img src="img/Comic.jpg" width="100px" height="100px" /><a href="">Comic</a>
```
[the remark this answers is not in the document]
Comment (AbbasShaikh0304) [reply tasks-by-users/AbbasShaikh0304/01-Project/index.html]: made alt changes @faran4engg

File: tasks-by-users/AbbasShaikh0304/01-Project/index.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    <h1>Online Books Store</h1>
+
+    <p>
+        Welcome to <strong>online Book Store</strong>.We sell the Books of varies
+        genre at most affordable prize. You can also download the digital copy of
+        your Books.
+    </p>
+
+    <h2>Genre</h2>
```
[the remark this answers is not in the document]
Comment (AbbasShaikh0304) [reply tasks-by-users/AbbasShaikh0304/01-Project/index.html:25]: That Css file was add mistakenly, i removed that file @faran4engg

**Review by faran4engg — CHANGES_REQUESTED** [review]
add .vscode folder in .gitignore

**Review by faran4engg — CHANGES_REQUESTED** [review]
what is this readmme.md, remove if unnecessary

**Review by anask601 — APPROVED** [review]

**Review by faran4engg — CHANGES_REQUESTED** [review]
@AbbasShaikh0304 , is this PR intended to be closed ( NO MERGE )

**Comment by nehalskdev** [comment]
Good one

**Pull request closed without merging**